composer.json

```diff
@@ -17,7 +17,9 @@
 	},
 	"require": {
 		"php": "^5.6 || ^7",
-		"wp-media/phpunit": "^1.0"
+		"brain/monkey": "^2.0",
+		"mikey179/vfsstream": "^1.6",
```

Comment:
aren't those going to be loaded anyway by the `wp-media/phpunit` package? Or it's a safeguard in case the dependency is remove from the dependency?

Reply:
Safe guard. Yes.